v2_test.go

```diff
@@ -241,7 +243,6 @@ var validParseAndWriteV2Tests = []struct {
 			TransportProtocol: UNSPEC,
 			SourceAddr:        nil,
 			DestinationAddr:   nil,
-			rawTLVs:           fixtureTLV,
```

Comment:
Same here: IIUC only addresses are ignored and attributes like TLVs and Transport Protocol.